<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormShowCamera</class>
 <widget class="QWidget" name="FormShowCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>339</width>
     <height>325</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lName">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lImage">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbPlay">
        <property name="text">
         <string>Play</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/icones/icones/play_green.png</normaloff>:/icones/icones/play_green.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbStop">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/icones/icones/stop_red.png</normaloff>:/icones/icones/stop_red.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSnapshot">
        <property name="text">
         <string>Snapshot</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/icones/icones/camera.png</normaloff>:/icones/icones/camera.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="cbVertical">
          <property name="text">
           <string>Flip Vertical</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbHorizontal">
          <property name="text">
           <string>Flip Horizontal</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
